test/box/gh-5518-add-granularity-option.test.lua

```diff
@@ -0,0 +1,61 @@
+
+-- write data recover from latest snapshot
+env = require('test_run')
+test_run = env.new()
+
+test_run:cmd('create server test with script="box/gh-5518-add-granularity-option.lua"')
+-- Start server test with granularity == 2 failed (must be greater than or equal to 4)
+test_run:cmd('start server test with args="2" with crash_expected=True')
+-- Start server test with granularity == 7 failed (must be exponent of 2)
+test_run:cmd('start server test with args="7" with crash_expected=True')
+
+test_run:cmd('start server test with args="4"')
+test_run:cmd('switch test')
+s = box.schema.space.create('test')
+_ = s:create_index('test')
+
+-- Granularity determines not only alignment of objects,
+-- but also size of the objects in the pool. Thus, the greater
+-- the granularity, the greater the memory loss per one memory allocation,
+-- but tuples with different sizes are allocated from the same mempool,
+-- and we do not lose memory on the slabs, when we have highly distributed tuple sizes.
+-- This is somewhat similar to a large alloc factor
+
+-- Try to insert/delete to space, in case when UB sanitizer on,
+-- we check correct memory aligment
+for i = 1, 1000 do s:insert{i, i + 1} end
+box.slab.info()
+for i = 1, 1000 do s:delete{i} end
+s:drop()
+test_run:cmd('switch default')
+test_run:cmd('stop server test')
+
+test_run:cmd('start server test with args="64"')
+test_run:cmd('switch test')
+s = box.schema.space.create('test')
+_ = s:create_index('test')
+for i = 1, 1000 do s:insert{i, i + 1} end
+box.slab.info()
```

Comment:
Instead of dumping info content, could we compare stats explicitly (assert that box.slab.info() of instance with larger granularity greater than box.slab.info() of instance with smaller granularity)

Reply:
I would also check `box.slab.stats()` output. Better to show only `item_size` part of it.